$OpenBSD: patch-src_if-bsd_c,v 1.11 2020/04/03 13:34:36 sthen Exp $

Just after dhcpcd started using these constants(!), they were moved
from a kernel header to src/sbin/slaacd/engine.c. Just use dhcpcd's own
definitions instead.

Index: src/if-bsd.c
--- src/if-bsd.c.orig
+++ src/if-bsd.c
@@ -1728,14 +1728,14 @@ int
 ip6_temp_preferred_lifetime(__unused const char *ifname)
 {
 
-	return ND6_PRIV_PREFERRED_LIFETIME;
+	return TEMP_PREFERRED_LIFETIME;
 }
 
 int
 ip6_temp_valid_lifetime(__unused const char *ifname)
 {
 
-	return ND6_PRIV_VALID_LIFETIME;
+	return TEMP_VALID_LIFETIME;
 }
 
 #else /* __OpenBSD__ */
